lib/rules/no-constant-condition.js

```diff
@@ -120,12 +120,30 @@ module.exports = {
             return false;
         }
 
+        /**
+         * Checks if a identifier has been defined in this scope.
+         * @param {AstNode} node An identifier node to check.
+         * @returns {boolean} Is the identifier unresolved?
+         */
```

Comment:
```suggestion
        /**
         * Checks if an identifier is a reference to a global variable.
         * @param {ASTNode} node An identifier node to check.
         * @returns {boolean} `true` if the identifier is a reference to a global variable.
         */
```

I think these descriptions were for `identifierIsUnresolved`.

Reply:
Oops, thanks. Updated.